modules/final_radiological_review/php/NDB_Form_final_radiological_review.class.inc

```diff
@@ -171,6 +172,13 @@ class NDB_Form_final_radiological_review extends NDB_Form
                 );
             }
 
+            if ($DB->tableExists('radiology_review') === false) {
```

Comment:
Won't the code not make it here because of the select statement on line 150?

I think this check needs to be done above that, not after it.